main.py

```
@@ -1,6 +1,88 @@
+from tools.md_to_json.md_to_json import MarkdownToJsonTool
+from tools.md_to_latex.md_to_latex import MarkdownToLatexTool
 
-if __name__ == '__main__':
-    plugin.run()
+from dify_plugin.entities.tool import ToolInvokeMessage
+
+app = FastAPI()
+
+security = HTTPBearer()
+
+def authorize(credentials: HTTPAuthorizationCredentials = Depends(security)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)